lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1004,6 +1063,8 @@ def main():
         template=dict(default=None),
         template_version=dict(default=None, type='int'),
         use_latest_template_version=dict(default=None, type='bool'),
+        storage_domain=dict(default=None),
+        disk_format=dict(choices=['COW','RAW'], default='COW'),
```

Comment:
hmm, most of the things in other modules are in lower case, better to be consistent